README.md

```diff
@@ -1 +1,65 @@
-# Etch-a-Sketch-Tala
\ No newline at end of file
+# Etch-a-Sketch-Tala
+<br/>
+
+## Why I did this?
+In this project I'll be creating a pretty neat toy for my portfolio to flex my DOM manipulation skills. 
+
+
+## What is this page?
+* It's a browser version of something between a sketchpad and an Etch-A-Sketch.
+* It's used to draw any thing on a custom number of sqauers by hovering the mouse on them.
+* The user can insert any number of squares he wants.
+<br/>
+
+## How I did it?
```

Comment:
I believe this section has to mention the user journey, not technically what's going on under the hood from a programmer perceptive. 
for instance:
>  as a user, I would be able to draw on a grid by hovering onto the pixels..etc

whenever you add anew feature, you may add its own user journey.
PS: make it only one sentence each, specific and simple.